.github/workflows/hardware-smoke.yml

```diff
@@ -104,7 +104,7 @@ jobs:
             --tag e2e \
             -p $BMC_BOARD --device-testing \
             --hardware-map /opt/tenstorrent/twister/hw-map.yml --west-flash \
-            -T ../tt-zephyr-platforms/app -ll DEBUG \
+            -T ../tt-zephyr-platforms/app \
```

Comment:
Let's drop the commit removing `-ll DEBUG`. I'll look at how to reduce the verbosity of these logs in a future PR, if that's ok?